gateway/api/domain/function/check_logs.py

```diff
@@ -47,15 +47,15 @@ def check_logs(logs: Union[str, None], job: Job) -> str:
             logs_size,
             max_mb,
         )
-        ratio = max_bytes / logs_size
-        new_length = max(1, int(len(logs) * ratio))
 
-        # truncate logs depending of the ratio
-        logs = logs[:new_length]
-        logs += (
-            "\nLogs exceeded maximum allowed size ("
+        # truncate logs discarding older
+        logs = logs[-max_bytes:]
+
+        logs = (
+            "[Logs exceeded maximum allowed size ("
             + str(max_mb)
-            + " MB) and could not be stored."
+            + " MB). Older logs are discarded.]\n"
```

Comment:
I am not sure if the message is easy to understand. The intention here is to say that we truncate the logs starting from the latest ones, right? Maybe you could say:

```suggestion
            + " MB). Logs have been truncated, discarding the oldest entries first.]\n"
```